config/sidekiq_scheduler.yml

```diff
@@ -287,12 +287,12 @@ VBADocuments::RunUnsuccessfulSubmissions:
   description: "Run VBADocuments::UploadProcessor for submissions that are stuck in uploaded status"
 
 VBADocuments::UploadRemover:
-  cron: "15,45 * * * * America/New_York"
+  cron: "*/2 * * * * America/New_York"
```

Comment:
At one point, this was actually running every 30 seconds, so I've updated it to be every 2 minutes, which seemed slightly more reasonable while still being timely in removing documents from S3.